docs/dev/git-cheat.md

```diff
@@ -0,0 +1,126 @@
+# Git cheat sheet for (busy) contributors
+
+## Requirement: DCO and signing commits
+
+Always DCO and sign your commits:
+
+```bash
+git -S -s -m "My commit"
+```
+
+Note: you do NOT need a gpg key to sign commits. You can just use your ssh key.
```

Comment:
I guess GPG signing is still preferred over SSH.

Reply:
Does GitHub's Web UI support verifying SSH signs?

Reply:
> Does GitHub's Web UI support verifying SSH signs?

I think so.
I am signing with my ssh key.

> I guess GPG signing is still preferred over SSH.

No idea.